### ISSUE TITLE
Input Lag or what?

### ISSUE THREAD

**Issue by alexlubi** [haxball/haxball-issues#1197]
Hello everyone. During playing Haxball at full screen or not, I have every 5 minutes a lag that doesn't affect ping or sth, but literally my player goes in one direction itself, without my command. This happens with every quantity of extrapolation that I use, like 20, 100, 90, 160 (during F1 mode session). This seems like an input lag or something that will freeze for one moment, and It resumes. I have tried everything, also to disable Frame Rate or VSYNC, but I didn't solve the problem.
I am using Google Chrome with 60 FPS, monitor with 1600x900 resolution and 60 HZ of frequence, Windows 10 OS version 2H02 (problem persisting also from previous versions, but now it's still unacceptable).

**Comment by basro** [comment]
What you describe could be caused by packet loss in your connection.

I would advise you to try and measure the packet loss in your connection.
This site seems to do just that: https://packetlosstest.com/
Keep in mind that packet loss can vary depending on where you are connecting to, it's possible that this site will report 0 packet loss but your route to the server you play in is still busted.

If you are playing over wifi and have the chance to connect with an ethernet cable instead try doing so.

**Comment by alexlubi** [comment]
Hi basro. Thanks for your response.
I have done a test with:
Packet Sizes: 1059 and 1075 Bytes 
Frequency: 36 Pings/Second 
Duration: 180 Seconds 
Acceptable Delay: 160 Milliseconds

The final result is that there is a little loss of packet, 0.2%, while there is 1.3% of late packets. I am using TIM brand fiber connection, with Ethernet cable on my computer. What can you say about this? Probably that 0.2% could be crucial? I am having issues only with Haxball.

**Comment by basro** [comment]
0.2% is less than I was expecting but it's not ideal (I get 0% with those same test parameters)
What was your average latency and average jitter? Jitter is also something that you should avoid as much as possible.

Unfortunately since you are already using an ethernet cable I don't really know of any other easy ways for you to improve your connection.

**Comment by alexlubi** [comment]
If you mean latency on rooms, it depends from hosts. I have a value that can be from 18 to 63 ping. If you mean my internet latency, I have around 21 ping per millisecond.
Jitter average how i can verify that?

**Comment by basro** [comment]
I meant the results from the same site you used for the packet loss tests, it also reports average ping and jitter.

**Comment by 9999OldSchool** [comment]
I don't think it's only you, I have that bug too. I have few bugs in the game, but it's just pointless to report these bugs because the owner of the game never cares about gameplay, all what he does care is about headless hosts (of course, I know this comment doesn't make sense with this topic, but still). 

No hard feelings, my buddy basro, just saying what I think. 

EDIT: most of these players thinks so, but I guess we don't have that power like you, basro. Less and less players are playing Big v4 because it's just pure cancer only because of extrapolation, more and more input delays. I don't know how we still manage to get at least 2k players every day, in summer 2018 there weren't even 1.2k players, kinda funny. I guess it's because they tried other maps that doesn't cause to use extrapolation, and futsal is way too more popular than other maps combined, dude, which is nice, but not most skilled gamemode which is kinda sad.

**Comment by alexlubi** [comment]
> I meant the results from the same site you used for the packet loss tests, it also reports average ping and jitter.

Results are these:
Average Latency: 129.88ms
Average Jitter: 4.07ms

Anyway, I have tried to play from different browsers, like Mozilla, Opera ecc ecc.... the problem is persisting in every browser, but in some browsers problem is less, and doesn't ruin my match, and in some browsers the problem is hard like with Google Chrome. Is there some ways for eliminating it completely?

**Comment by Wenamka** [comment]
> I don't think it's only you, I have that bug too. I have few bugs in the game, but it's just pointless to report these bugs because the owner of the game never cares about gameplay, all what he does care is about headless hosts (of course, I know this comment doesn't make sense with this topic, but still).
> 
> No hard feelings, my buddy basro, just saying what I think.
> 
> EDIT: most of these players thinks so, but I guess we don't have that power like you, basro. Less and less players are playing Big v4 because it's just pure cancer only because of extrapolation, more and more input delays. I don't know how we still manage to get at least 2k players every day, in summer 2018 there weren't even 1.2k players, kinda funny. I guess it's because they tried other maps that doesn't cause to use extrapolation, and futsal is way too more popular than other maps combined, dude, which is nice, but not most skilled gamemode which is kinda sad.

tbh you should make 2 def or 1 def rules for your trash format, basro made "big easy" map for it in 2011, ofc youll lose your community isnt about extrapolation, its about promotion of game, no one of new generation knows about Haxball and thats logic what old players retire and stopped with haxball but wheres new players? haxball was popular 8-9 years ago, so someone still playing from that time but to 2025 numbers of players will be like 200 turkish players and 100 players rest of out world.
Basro knows it and wont makes something for promote game, advertisement in other games or steam joins, he dont care even about something of new features in game

**Comment by 9999OldSchool** [comment]
Yes, I agree. It is what it is, it had biggest potential to be even e-sports game, sadly.

**Comment by Jacob-1998** [comment]
Basro why dont you start taking bids for this game already? You can even keep a 20% share, sell 80% only and then if the game blows up thanks to a more competent owner making use of Haxball's potential, you still get rich.
